src/GUI/SimpleJavaFXGUI.java

```diff
@@ -0,0 +1,64 @@
+package GUI;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Node;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.input.MouseEvent;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+
+public class SimpleJavaFXGUI extends Application {
+
```

Comment:
voeg in comments de functies toe met een korte beschrijving(zie de database package van marit voor een voorbeeld). geef hier ook aan wat de functies accepteren als input en wat ze teruggeven.